src/api/OrganizationsApi.js

```diff
@@ -571,6 +573,36 @@ function getOrganization(organizationId :UUID) :Promise<*> {
     });
 }
 
+/**
+ * `GET /organizations/{orgId}/datasource`
+ *
+ * @static
+ * @memberof lattice.OrganizationsApi
+ * @param {UUID} organizationId
+ * @returns {Promise<Object[]>} - a Promise that resolves with the organization data sources
+ *
+ * @example
+ * OrganizationsApi.getOrganizationDataSources("ec6865e6-e60e-424b-a071-6a9c1603d735");
+ */
+function getOrganizationDataSources(organizationId :UUID) :Promise<Object[]> {
```

Comment:
datasource is one word

Reply:
also does this API exist? I'm not seeing it

Reply:
Matthew is implementing these in the `feature/pluggable-warehouse-support` branch.

https://github.com/openlattice/openlattice/tree/feature/pluggable-warehouse-support

Reply:
Sticking with `DataSource` based on our conversation in slack.